package.json

```diff
@@ -13,7 +13,7 @@
     "clobber": "rm -rf elm-stuff demo/elm-stuff demo/.webpack tests/elm-stuff tests/Doc",
     "rebuild": "npm run clobber && npm install && npm test",
     "deploy:demo": "cd demo && cross-env NODE_ENV=production serverless deploy",
-    "postinstall": "elm package install -y && cd demo && elm package install -y",
+    "postinstall": "elm package install -y",
```

Comment:
I tried removing this line altogether, and removing all the `elm-stuff` directories, then `npm test` and it worked. I guess running `elm package install` is not necessary. Might be the webpack plugin takes care of installing things.

Can you update the PR to just remove this line?